rel/overlay/etc/default.ini

```diff
@@ -443,16 +443,25 @@ level = info
 ; max_message_size = 16000
 ;
 ;
-; There are three different log writers that can be configured
+; There are four different log writers that can be configured
 ; to write log messages. The default writes to stderr of the
 ; Erlang VM which is useful for debugging/development as well
 ; as a lot of container deployments.
 ;
-; There's also a file writer that works with logrotate and an
+; There's also a file writer that works with logrotate, a
 ; rsyslog writer for deployments that need to have logs sent
```

Comment:
ps. should this say `syslog` to map directly to the writer module name?

Reply:
not a bad idea :)